scipy/stats/stats.py

```diff
@@ -82,6 +82,9 @@
     obrientransform
     signaltonoise
     sem
+    zmap
+    zscore
+    iqr
```

Comment:
this addition is fine, but the more important one should be made in the function listing in ``stats/__init__.py``